ebu_tt_live/documents/test/test_ebutt3deduplication.py

```diff
@@ -0,0 +1,198 @@
+from unittest import TestCase
```

Comment:
Naming of file is actually not consistent with rest of repo it should be `test_ebutt_3_deduplication.py`